odoo/tools/translate.py

```diff
@@ -521,7 +521,16 @@ def __call__(self, source, *args, **kwargs):
     def _get_translation(self, source, module=None):
         try:
             frame = inspect.currentframe().f_back.f_back
-            lang = self._get_lang(frame)
+            translation_file = inspect.getfile(frame)
```

Comment:
This is not the `translate.py` file. You're already going two stackframes up, so you're in the file and stackframe of the caller.

Reply:
You are right! I misunderstood it.